docker-compose-installed.yaml

```diff
@@ -109,6 +109,8 @@ services:
   devmode-rust:
     image: hyperledger/sawtooth-devmode-engine-rust:nightly
     container_name: devmode-engine-rust
+    ports:
```

Comment:
I think this is unnecessary: it binds the devmode-rust container's port 5050 to the host's port 5050.  What was missing is the `--bind consensus...` CLI arg

Reply:
Accepted. It didn't seem to work until I added port 5050 to the devmode container.
I removed it and it still works, so I rebased and pushed this.